pygdf/categorical.py

```diff
@@ -155,6 +155,52 @@ def to_pandas(self, index=None):
                                          ordered=self._ordered)
         return pd.Series(data, index=index)
 
+    def to_pandas(self, index=None):
+        return pd.Series(self.to_array(fillna='pandas'), index=index)
```

Comment:
This function is already defined above this. Also this would return a Pandas Series of objects as opposed to a categorical. I think you can remove these two lines.